$OpenBSD: patch-__init___py,v 1.1.1.1 2013/04/24 18:42:20 zhuk Exp $
We do not have libdl here, on OpenBSD. importing DLFCN
should be optional, but my py-fu is not good enough.
--- __init__.py.orig	Fri Oct 14 04:25:47 2011
+++ __init__.py	Fri Oct 14 04:40:18 2011
@@ -1,4 +1,4 @@
-import sys,DLFCN
+# import sys,DLFCN
 # This is needed to ensure that dynamic_cast and RTTI works inside kdelibs.
-sys.setdlopenflags(DLFCN.RTLD_NOW|DLFCN.RTLD_GLOBAL)
-     
\ No newline at end of file
+# sys.setdlopenflags(DLFCN.RTLD_NOW|DLFCN.RTLD_GLOBAL)
+     
